js/web/lib/wasm/jsep/webgpu/ops/layer-norm.ts

```diff
@@ -55,40 +49,44 @@ const createLayerNormProgramInfo =
           }
 
           const dataType = tensorTypeToWsglStorageType(inputs[0].dataType);
+          const components = getMaxComponents(normSize);
+          const variables = [
+            inputVariable('x', inputs[0].dataType, inputs[0].dims, components),
+            inputVariable('scale', scale.dataType, scale.dims, components),
+          ];
+          if (bias) {
+            variables.push(inputVariable('bias', bias.dataType, bias.dims, components));
+          }
+          variables.push(outputVariable('output', inputs[0].dataType, outputShape, components));
 
           const hasMeanDataOutput = outputCount > 1;
           const hasInvStdOutput = outputCount > 2;
-          let bindingIndex = 0;
+
+          if (hasMeanDataOutput) {
```

Comment:
Should we also use `DataType.float` for `Mean` output and `InvStdDev` output for `LayerNormalization` and cast the corresponding data to f32 (`CastToF32`) like you did in `SkipLayerNormalization`?

Reply:
Yeah, I've missed that